docker-files/base/docker-bootstrap.sh

```diff
@@ -8,12 +8,12 @@ do
       sphinx \
       nodejs6 npm6 \
       mariadb-client \
+      mysql-devel \
```

Comment:
So. Do we need mysql-devel then?

Reply:
Wouldn't it be `libmariadb-devel`?

Reply:
We need it to update mysql2 gem on circleci - in case depfu comes with a PR.

And libmariadb-devel gets installed if you zypper in mysql-devel - but you're right, for leap 42 libmariadb-devel would be the proper choice